db/migrate/20180914132709_convert_accepting_to_enum.rb

```diff
@@ -0,0 +1,35 @@
+class ConvertAcceptingToEnum < ActiveRecord::Migration[5.1]
+  def up
+    # NOTE: Wanted to RAISE a DB error on unknown value,
+    #       but seems Rails/AR doesn't support that
+    # ... ELSE RAISE EXCEPTION 'UNKNOWN ACCEPTING VALUE IN ROW (ID %): %', id, accepting
+    execute <<-DDL
+      CREATE TYPE shelter_accepting_value AS ENUM (
```

Comment:
This breaks dumping the shelters table to schema.rb (see that file below).  Either don't use an enum in the database, or update the configuration in "config/application.rb" with `config.active_record.schema_format = :sql` so that it exports the database schema as an SQL dump at "db/structure.sql" instead.